core/tests/webdriverio_utils/ExplorationEditorMainTab.js

```diff
@@ -439,8 +440,13 @@ var ExplorationEditorMainTab = function() {
 
     await editOutcomeDestDropdownOptions.selectByVisibleText(targetOption);
 
+    // 'End' is one of the keys name for setValue function, so to insert
+    // 'End' as a string we need to pass it as an array of string.
+    if (destName === 'End') {
+      destName = ['E', 'n', 'd'];
+    }
     if (createNewDest) {
-      var editOutcomeDestStateInput = editOutcomeDestBubble.$(
+      var editOutcomeDestStateInput = editOutcomeDestForm.$(
```

Comment:
Why this was changed?

Reply:
Chain selectors only work in the consecutive elements so instead of using detBubble I am using chain selectors with the immediate parent of the selected element.